tests/integration_tests/core_tests.py

```diff
@@ -429,6 +429,7 @@ def test_slices(self):
             self.assertEqual(resp.status_code, 200)
 
     def test_tablemodelview_list(self):
+        pytest.skip("Depends on ENABLE_REACT_CRUD_VIEWS=False")
```

Comment:
I'm skipping these and requesting QA to help with replacing the lost test coverage.

Reply:
I don't think there's any real loss of coverage here. `ENABLE_REACT_CRUD_VIEWS=True` has been the case for most large deployment of superset so far, so these test are just covering functionality that no one is using. The api tests and js tests provide decent coverage for the new crud view, imo.